actionpack/lib/action_controller/template_assertions.rb

```diff
@@ -102,6 +102,11 @@ def reset_template_assertion
     #   # assert that the "_customer" partial was rendered with a specific object
     #   assert_template partial: '_customer', locals: { customer: @customer }
     def assert_template(options = {}, message = nil)
+      ActiveSupport::Deprecation.warn(
```

Comment:
Do you want me to transfer the Gem to Rails? Although Rails isn't supporting these two methods, it seems weird that we're pointing to my personal repository.